io/disk/rawread.py.data/rawread.yaml

```diff
@@ -0,0 +1,10 @@
+# Bonnie options:
+# bonnie++ [-d scratch-dir] [-c concurrency] [-s size(MiB)[:chunk-size(b)]]
+#       [-n number-to-stat[:max-size[:min-size][:num-directories[:chunk-size]]]]
+#       [-m machine-name] [-r ram-size-in-MiB]
+#       [-x number-of-tests] [-u uid-to-use:gid-to-use] [-g gid-to-use]
+#       [-q] [-f] [-b] [-p processes | -y] [-z seed | -Z random-file]
+#       [-D]
```

Comment:
Please correct it for rawread

Reply:
looks like I have attached the wrong file.
I have the edited file. will attach it. Thanks for it.